### ISSUE TITLE
size of order list in settings is fixed

### ISSUE THREAD

**Issue by Rudd-O** [dfaust/plasma-applet-popup-launcher#4]
I expected that resizing the window would resize the size of the order list.  It does not.  It's painful to sort things with it because of its small size.

**Comment by dfaust** [comment]
Yeah, the whole config dialog is more of a hack right now, but I don't have the time to fix this. So this will be a low priority for me.
